components/Item/ItemMenu.vue

```diff
@@ -56,9 +63,42 @@ export default Vue.extend({
             }
           });
         }
+
+        if (
+          this.$auth.user?.Policy?.IsAdministrator &&
+          (this.item.Type === 'Folder' || this.item.Type === 'CollectionFolder')
```

Comment:
```suggestion
          (['Folder', 'CollectionFolder', 'UserView'].includes(this.item.Type || '')
```

`UserView` is for merged libraries (In jf-web, go to Settings > Home > Library Folders)

Reply:
Ah ok, updated 👍